pkg/operator/target_config_reconciler.go

```diff
@@ -417,7 +172,6 @@ func (c *TargetConfigReconciler) manageDeployment(descheduler *deschedulerv1beta
 			UID:        descheduler.UID,
 		},
 	}
-	required.Spec.Template.Spec.Containers[0].Image = descheduler.Spec.Image
```

Comment:
Are not we using this to set the actual descheduler image? Without it, the image gets set to the upstream one.

Reply:
No we aren't going to allow custom descheduler images through the operator, but I did just push a new commit to default to our quay.io/openshift image